src/open_inwoner/templates/pages/profile/contacts/list.html

```diff
@@ -8,6 +8,47 @@ <h1 class="h1" id="title">
     {% button href="accounts:contact_create" text=_("Nieuw contact toevoegen") icon="person" primary=True %}
 </h1>
 
+<!--List of contact requests: receiver's view-->
+{% if pending_approval or pending_approval_list %}
+    <div class="approval{% if pending_approval %} approval--solo{% endif %}">
+        <div class="approval__header">
+            <h2 class="h2">{% trans "U bent toegevoegd als contactpersoon" %}</h2>
+            <p class="p">{% trans "By accepting you agree on sharing your personal data (first name, last name and phonenumber)." %}</p>
+        </div>
+        {% if pending_approval %}
+            <div class="approval__single">
+                <p class="p">
+                    {% blocktrans with full_name=pending_approval.get_full_name %}{{ full_name }} heeft u toegevoegd aan diens contactpersonen.{% endblocktrans %}
+                </p>
+                {% url 'accounts:contact_approval' uuid=pending_approval.uuid as approval_url %}
+                {% render_form form=None method="POST" form_action=approval_url id="approval_form" spaceless=True show_notifications=True %}
+                    {% csrf_token %}
+                    {% button_row %}
+                        {% button text=_("Weigeren") name="contact_reject" value="reject" secondary=True bordered=True %}
```

Comment:
I don't think the value is needed here. I think the button name is enough (and in the view you can just check something like `"contact_reject" in request.POST`